generator/pom.xml

```diff
@@ -46,7 +46,7 @@
         <dependency>
             <groupId>org.apache.commons</groupId>
             <artifactId>commons-lang3</artifactId>
-            <version>3.12.0</version>
+            <version>${commons-lang3.version}</version>
```

Comment:
Does this mean that the version used by the generator and the SDK must be the same now?
I don't mind, we can still split it future if needed, right?

Reply:
They don't have to match per se but it makes sense to keep it this way if there were to be any CVEs in one version fixed in another.